picamera2/devices/imx708/imx708.py

```diff
@@ -0,0 +1,46 @@
+import fcntl
+import os
+
+from picamera2 import Picamera2
+from v4l2 import *
+
+HDR_CTRL_ID = 0x009a0915
+
+
+class IMX708:
+    def __init__(self, camera_num: int):
+        self.device_fd = None
+
+        camera_id = next((c['Id'] for c in Picamera2.global_camera_info() if c['Num'] == camera_num), None)
+        if camera_id is None:
+            raise RuntimeError(f'IMX708: Requested camera number {camera_num} cound not be found')
```

Comment:
Should we be checking that we do indeed have an imx708? Or selecting it automatically if there's just one of them? Not sure...

Reply:
> Should we be checking that we do indeed have an imx708?

This is done when traversing the v4l2 subdev nodes below. 

> Or selecting it automatically if there's just one of them?

I'll add this so that camera_num can be optional and it selects the first imx708 sensor found.